pkg/controller/config/config_controller.go

```diff
@@ -231,9 +231,9 @@ func (r *ReconcileConfig) Reconcile(req reconcile.Request) (reconcile.Result, er
 		return r.applyPipeline(req, cfg)
 	case op.AppliedPipeline, op.PipelineValidateError:
 		return r.validatePipeline(req, cfg)
-	case op.ValidatedPipeline, op.AddonsError:
+	case op.ValidatedPipeline, op.AddonsError, op.AppliedPipelineVersion:
 		return r.applyAddons(req, cfg)
-	case op.AppliedAddons, op.CommunityResourcesError:
+	case op.AppliedAddons, op.CommunityResourcesError, op.AppliedTriggerVersion:
```

Comment:
could we reuse the existing states, `ValidatedPipelines` and `AppliedAddons` 
to handle the new additions

Reply:
we need status to add to the array

Reply:
removed the new status